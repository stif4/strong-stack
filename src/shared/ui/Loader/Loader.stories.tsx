import React from 'react';
import { Meta, StoryFn } from '@storybook/react';
import { Loader } from './Loader';

export default {
    title: 'shared/Loader',
    component: Loader,
} satisfies Meta<typeof Loader>;

const Template: StoryFn<typeof Loader> = (args) => <Loader {...args} />;

export const Normal = Template.bind({});
Normal.args = {};
