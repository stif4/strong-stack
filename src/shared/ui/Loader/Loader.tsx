import { classNames } from 'shared/helpers/classNames/classNames';
import './Loader.scss';

interface LoaderProps {
    className?: string;
}

export const Loader = ({ className }: LoaderProps) => (
    <div className={classNames('ldsEllipsis', {}, [className])}>
        <div />
        <div />
        <div />
        <div />
    </div>
);
