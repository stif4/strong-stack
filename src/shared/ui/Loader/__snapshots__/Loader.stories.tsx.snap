// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/Loader Normal smoke-test 1`] = `
<div class="app dark">
  <div class="ldsEllipsis">
    <div>
    </div>
    <div>
    </div>
    <div>
    </div>
    <div>
    </div>
  </div>
</div>
`;
